lifecycle-operator/config/samples/lifecycle_v1alpha4_keptnworkloadversion.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: lifecycle.keptn.sh/v1alpha4
+kind: KeptnWorkloadVersion
+metadata:
+  annotations:
+    traceparent: my-trace-parent
+  name: some-keptn-workload-instance-2
```

nitpick, just for consistency
```suggestion
  name: some-keptn-workload-version
```